cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -43,3 +50,47 @@ def test_awsbatch_cluster_builder(mocker):
     )
     print(yaml.dump(generated_template))
     # TODO assert content of the template by matching expected template
+
+
+@pytest.mark.parametrize(
+    "config_file_name, expected_head_node_dna_json_file_name",
+    [
+        ("slurm-imds-secured-true.yaml", "slurm-imds-secured-true.head-node.dna.json"),
+        ("slurm-imds-secured-false.yaml", "slurm-imds-secured-false.head-node.dna.json"),
+        ("awsbatch-imds-secured-false.yaml", "awsbatch-imds-secured-false.head-node.dna.json"),
+    ],
+)
+@freeze_time("2021-01-01T01:01:01")
```

Comment:
can you add a comment explaining why time is frozen at this particular value?

Reply:
There is no reason for that: it simply matches the value put into the yaml to check.

Reply:
There is no reason for that: it simply matches the value put into the yaml to check.

Reply:
i was talking about the general freezing of time rather than the specific time at which we froze it, because if i were unfamiliar with what's going on with this test i think it would be difficult to understand why time needed to be frozen. either way it's a minor thing.

Reply:
> i was talking about the general freezing of time rather than the specific time

Ha, sorry I'm now looking at my first comment and I said the exact opposite.

Reply:
Ok, I'll add a short comment to clarify it.